src/style.css

```diff
@@ -1 +1,77 @@
 /* Styles go here */
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 14px;
+  box-sizing: border-box;
+}
+
+.header {
+  max-width: 1200px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  margin: 0 auto;
+}
+
+.img__logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.list__item {
+  text-transform: uppercase;
+  color: #000;
+  margin-right: 20px;
+}
+
+.list__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  position: relative;
+  display: inline-block;
+  height: 60px;
+  line-height: 60px;
+  text-decoration: none;
+  color: #000;
+  cursor: pointer;
+}
+
+.nav__link:hover {
+  color: hsl(193, 100%, 43%);
+}
+
+.is-active {
+  color: hsl(193, 100%, 43%);
+}
+
+.is-active::after {
+  position: absolute;
+  content: "";
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  background-color: hsl(193, 100%, 43%);
+  border-radius: 8px;
+}
+
+@media (max-width: 1024px) {
```

Comment:
in requirements you have one more cases with 1200
<img width="836" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/6568745/1c76bacc-ecfe-409d-8dcf-1a9fbe8df286">

Reply:
dont understand what i need to do, there is in 
.header {
  max-width: 1200px;

Reply:
you are correct